src/Entities/EnderCrystal.cpp

```diff
@@ -47,11 +88,9 @@ void cEnderCrystal::Tick(std::chrono::milliseconds a_Dt, cChunk & a_Chunk)
 void cEnderCrystal::KilledBy(TakeDamageInfo & a_TDI)
 {
 	Super::KilledBy(a_TDI);
-
-	m_World->DoExplosionAt(6.0, GetPosX(), GetPosY(), GetPosZ(), true, esEnderCrystal, this);
-
 	Destroy();
 
+	m_World->DoExplosionAt(6.0, GetPosX(), GetPosY() + (GetHeight() / 2.0), GetPosZ(), true, esEnderCrystal, this);
```

Comment:
What's this change for? Using `this` after `Destroy` seems sketchy.

Reply:
A bit, yeah- But i tried the other way around and way to many  exlosions were triggered

Reply:
What do you mean? The crystal exploded multiple times?

Reply:
Like one explosion triggering another...
Tested again. Doesn't happen anymore

Reply:
Destroy only queues.